src/main/java/Duke.java

```diff
@@ -1,10 +1,315 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
+    /** The Logo of Duke */
+    private static final String MSG_LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /** Line to be appended before and after a message */
+    private static final String MSG_LINE = "____________________________________________________________\n";
+
+    /** Message to be printed before to greet the user */
+    private static final String[] MSG_GREET = {"Hello! I'm Duke", "What can I do for you?"};
+
+    /** Message to be printed before exiting */
+    private static final String MSG_EXIT = "Bye. Hope to see you again soon!";
+
+    /** Message to be printed after marking task as done */
+    private static final String MSG_MARK = "Nice! I've marked this task as done:";
+
+    /** Message to be printed after changing status of task to not done */
+    private static final String MSG_UNMARK = "OK, I've marked this task as not done yet:";
+
+    /** Message to be printed after adding task to list of tasks */
+    private static final String[] MSG_ADD = {"Got it. I've added this task:", "Now you have ", " tasks in the list."};
+
+    /** Message to be printed after adding task to list of tasks */
+    private static final String MSG_DELETE = "Noted. I've removed this task:";
+
+    /** Indentation level of messages to be printed to console */
+    private static final int INDENTATION_LEVEL = 4;
+
+    protected enum Command {TODO, DEADLINE, EVENT, MARK, UNMARK, DELETE, LIST, BYE};
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        // Greets the user
+        try {
+            greet();
+        } catch (DukeException e) {
+            echo(e.getMessage());
+        }
+
+        // Initialise variables
+        ArrayList<Task> tasks = new ArrayList<>();
+
+        // Decision loop
+        boolean isContinue_decisionLoop = true;
```

Comment:
Maybe name the variable in camel case? Since it is not a method used in testing.